lib/plane.rb

```
@@ -0,0 +1,27 @@
+class Plane
+	attr_accessor :current_airport
+
+	def initialize
+		@current_airport
+	end
+
+	def land(airport)
+		if airport.weather == "stormy"
+			"Stormy weather. Unable to land."
+		elsif airport.landed_planes.count >= airport.capacity
```

Avoid using string return values to indicate success/failure as they can be easily changed.  Better to raise an exception in this case as we're only interested in a message if something goes wrong.